carbonserver/carbonserver/api/services/signup_service.py

```diff
@@ -38,5 +44,31 @@ def subscribe_user_to_org(
             organization_id, organization_api_key
         )
         if key_is_valid:
-            self._user_repository.subscribe_user_to_org(user, organization_id)
+            user = self._user_repository.subscribe_user_to_org(user, organization_id)
         return user
+
+    def new_user_steps(self, user: User) -> User:
```

Comment:
can this method be renamed into something more explicit ?
I have trouble naming this method, but here are some suggestions : 
- initialize_org_and_default_project
- register_user_with_default_project

Reply:
`new_user_setup` ?

Reply:
I am more in favor of `new_user_setup`, because we don't know what other steps we might have to do in the future to a new user. What do you think @SaboniAmine ?

Reply:
Yes, lgtm !